modules/persistent-storage-csi-vol-detach-non-graceful-shutdown-procedure.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-vol-detach-non-graceful-shutdown-procedure_{context}"]
+= Adding an out-of-service taint manually for automatic volume detachment
+
+.Prerequisites
+
+* Access to the cluster with cluster-admin privileges.
+
+.Procedure
+
+To allow volumes to detach automatically from a node after a non-graceful node shutdown:
+
+. After a node is detected as unhealthy, shut down the worker node.
```

Comment:
The node is supposed to be shutdown. Here we make to make sure that the node is indeed shutdown (and not hanging, rebooting or so)

Reply:
@gcharot isn't that ensured by asking the user to shut down the node? If it were hanging/rebooting, ti will indeed be shutdown when the user explicitly shuts it down, right?

Reply:
yes, it's a small nuance, the node is supposed to be shutdown what we ask is to ensure it is actually shutdown